examples/activation_and_marker_tracking/activation_tracker.py

```diff
@@ -0,0 +1,219 @@
+from scipy.integrate import solve_ivp
+import numpy as np
+import biorbd
+from casadi import MX, Function
+from matplotlib import pyplot as plt
+
+from biorbd_optim import OptimalControlProgram
+from biorbd_optim.mapping import Mapping
+from biorbd_optim.dynamics import Dynamics
+from biorbd_optim.plot import PlotOcp
+from biorbd_optim.problem_type import ProblemType
+from biorbd_optim.objective_functions import ObjectiveFunction
+from biorbd_optim.path_conditions import Bounds, QAndQDotBounds, InitialConditions
+
+
+def generate_data(biorbd_model, final_time, nb_shooting):
```

Comment:
For example, having a variable here (or in the original file) that reads: `muscle_traking_type` that can either take `"excitation"` or `"activation"` as argument